kinder/pkg/extract/extract.go

```diff
@@ -482,14 +484,37 @@ func saveVersionFile(addVersionFileToDst bool, dst string, version *K8sVersion.V
 	return nil
 }
 
+// Exponential backoff for httpGet (values exclude jitter):
+// 1, 2, 4, 8, 16, 32, 64, 128 s
+var httpGetBackoff = wait.Backoff{
+	Steps:    8,
+	Duration: 1000 * time.Millisecond,
+	Factor:   2.0,
+	Jitter:   0.1,
+}
+
 func httpGet(uri string) (int64, io.ReadCloser, error) {
-	resp, err := http.Get(uri)
+	var lastError error
+	var resp *http.Response
+	err := wait.ExponentialBackoff(httpGetBackoff, func() (bool, error) {
```

Comment:
as mentioned on slack, i would have preferred the other option to use `ci/latest-bazel` temporary.
this will mask the current k8s build problems.

yet the retry is a nice to have in general and we ultimately care about tests passing.
from my experiments ~60 seconds should be enough, but let's see how this goes.

/lgtm